packages/universal-swap/src/msg/types.ts

```
@@ -0,0 +1,36 @@
+}
+
+export enum PostActionType {
+  None,
+  Transfer,
+  ContractCall,
+  IbcTransfer,
+  IbcWasmTransfer
+}
+
+export interface MiddleWareResponse {
+  memo: String;
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)